### ISSUE TITLE
[Color Scheme - Camphell]: Name is not defined for 'Delete Color Scheme' button.

### ISSUE THREAD

**Issue by Priyanka-Chauhan123** [microsoft/terminal#15984]
### Windows Terminal version

1.19.2481.0

### Windows build number

23529.1000

### Other Software

Test Environment:
OS: Windows 11 Version Dev (OS Build 23529.1000)
App: Windows Terminal Preview

### Steps to reproduce

**Repro Steps: **

1.Open Windows Terminal.
2.Open Settings page using 'Ctr+,'
3. Navigate to the 'Color Scheme' and activate 'Add new'.
4. Now navigate to 'Delete color scheme' button and observe the issue.

**User Experience: **
Screen Reader users will not know the exact reference of the control, if name is not defined for the 'Delete color scheme' button.

MAS Reference:
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Attachment:

[Name is not defined for 'Delete Color Scheme' button.zip](https://github.com/microsoft/terminal/files/12644705/Name.is.not.defined.for.Delete.Color.Scheme.button.zip)

### Expected Behavior

Name should be defined for 'Delete Color Scheme' button.

### Actual Behavior

Name is not defined for 'Delete Color Scheme' button.
